assets/scss/components/_cart.scss

```diff
@@ -1,7 +1,22 @@
+.c-cart__wrapper {
+    display: grid;
+    position: relative;
+    min-height: fit-content;
+
+    @include bp(medium) {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
 .c-cart__totals {
     display: flex;
     flex-direction: column;
     align-items: flex-end;
+    position: sticky;
+    position: -webkit-sticky;
+    top: 20px;
+    height: min-content;
+    z-index: 10;
```

Comment:
Consider putting the `z-index` into variable. It would make it easier to maintain all z-indexes in the future :)

Reply:
I actually removed it, we don't need it anymore :)